config/routes.rb

```diff
@@ -1,5 +1,5 @@
 Rails.application.routes.draw do
-  constraints defaults: { format: :json } do
+  constraints do
```

Comment:
można wywalić cały blok `constraints`; sprawdź czy to będzie działać, ale raczej nie jest (i nie był nigdy) potrzebny